Makefile

```
@@ -35,6 +36,34 @@ setup-githooks:
 setup: setup-githooks
 	dep ensure
 
+.PHONY: debug-setup
+debug-setup:
+	@echo Exporting env vars to run operator locally:
+	- . ./scripts/export_local_envvars.sh
```

```suggestion
	- source ./scripts/export_local_envvars.sh
```
(just to be more explicit, and use the longer form (`source`) instead of the shorter form (`.`)

---

It will not work. Following the error.

```
make: source: No such file or directory
make: [debug-setup] Error 1 (ignored)
```

---

Can we close this one? Are you ok with?